spec/linter-erlc-spec.js

```diff
@@ -0,0 +1,68 @@
+'use babel';
+
+// eslint-disable-next-line no-unused-vars
+import { it, fit, wait, beforeEach, afterEach } from 'jasmine-fix';
+import * as path from 'path';
+import linter from '../lib/init';
+
+const goodPath = path.join(__dirname, 'fixtures', 'good.erl');
+const badPath = path.join(__dirname, 'fixtures', 'bad.erl');
+const emptyPath = path.join(__dirname, 'fixtures', 'empty.erl');
+
+describe('The ERLC provider for Linter', () => {
+  const lint = linter.provideLinter().lint;
+
+  beforeEach(async () => {
+    await atom.packages.activatePackage('linter-erlc');
+  });
+
+  it('should be in the packages list', () =>
+    expect(atom.packages.isPackageLoaded('linter-erlc')).toBe(true),
+  );
+
+  it('should be an active package', () =>
+    expect(atom.packages.isPackageActive('linter-erlc')).toBe(true),
+  );
+
+  it('finds nothing wrong with a valid file', async () => {
+    const editor = await atom.workspace.open(goodPath);
+    const messages = await lint(editor);
+    expect(messages.length).toBe(0);
+  });
+
+  describe('shows errors in a file with issues', () => {
+    let editor = null;
+
+    beforeEach(async () => {
```

Comment:
Since there is only one `it` in this `describe` block there is no need for this `beforeEach`, just merge it into the `it` like the block above.